### PR TITLE
Update meta file for daily sync testing

### PR DESCRIPTION
**Pull request by ShammiL**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by ChinthakaJ98 — APPROVED** [review]

File: .connector-store/meta.json [hunk 1]
```diff
@@ -19,6 +23,28 @@
                 "MI 4.1.0",
                 "MI 4.0.0"
             ],
+            "operations": [
+                {
+                    "name": "init",
+                    "description": "configure Redis connector",
+                    "isHidden": false
```
Comment (chanukaranaba) [inline .connector-store/meta.json:30]: @ShammiL can this operations could be specific to a particular version? 
Just as an example lets say ping method is not supporting in a future version. How should we support it? isn't it better add the version and the product in the operations.

Comment (shayanmalinda) [reply .connector-store/meta.json:30]: Looping in @ChinthakaJ98

Comment (shayanmalinda) [reply .connector-store/meta.json:30]: +1 to add operations against a release

Comment (ChinthakaJ98) [reply .connector-store/meta.json:30]: In this approach the operations are added against the release. As per this metafile the given operations are only valid for the 3.1.3 version. If there is another release the relevant operations should be added separately for that.

Comment (chanukaranaba) [reply .connector-store/meta.json:30]: Ack.

**Pull request merged**